### PR TITLE
fixes #320

### LINKED ISSUE #320: ALTER TABLE TB1 ADD CONSTRAINT C1 UNIQUE (COL1, COL2) not supported
The following SQLstatement is not supported by the latest version of jSqlParser:
`ALTER TABLE TB1 ADD CONSTRAINT C1 UNIQUE (COL1, COL2)`

The following variation works (Issue  #105 talks about this syntax)
`ALTER TABLE TB1 ADD UNIQUE KEY C1 (COL1, COL2)`

This form of `ALTER` statement is generated by EclipseLink (`Persistence.generateSchema(..)`) with a DB2 dialect
Could it be possible to add this syntax to jSqlParser?
Thanks

### PR DESCRIPTION
**Pull request by tmnsur**
Possible fix for #320

### CODE REVIEW

**Comment by wumpz** [comment]
You need to test the deparsing. That should give you the original sqlTxt. meaning the deparser should rebuild the input syntax. You could use for the the tool funktion assertThatSqlCanBeParsedAndDeparsed.

**Comment by tmnsur** [comment]
it was already in 91fcb4c, between some more detailed checks. In last commit (9bdd4d2) i removed all lines but assertThatSqlCanBeParsedAndDeparsed.

**Comment by wumpz** [comment]
Good. To merge I always use squash and merge from github. This would create a new summarizing commit from your commits. Therefore you would loose the connection to your branch. If thats ok for you, we could proceed. But if you are not, then provide a pull request with one commit.

**Comment by tmnsur** [comment]
That's ok, np.

File: src/test/java/net/sf/jsqlparser/test/alter/AlterTest.java [hunk 1]
```diff
@@ -252,4 +253,28 @@ public void testDropColumnRestrictIssue551() throws JSQLParserException {
...
+        String tableName = "table1";
+        String columnName1 = "col1";
+        String columnName2 = "col2";
+        String columnName3 = "col3";
+        String columnName4 = "col4";
+        String constraintName1 = "table1_constraint_1";
+        String constraintName2 = "table1_constraint_2";
+
+        for(String constraintType : Arrays.asList("UNIQUE KEY", "KEY")) {
+            Statement stmt = CCJSqlParserUtil.parse("ALTER TABLE " + tableName + " ADD CONSTRAINT " + constraintName1 + " " + constraintType + " (" + columnName1 + ")");
```
Comment (wumpz) [inline src/test/java/net/sf/jsqlparser/test/alter/AlterTest.java]: I merged and changed now this lines. You used assertStatementCanBeDeparsedAs which needs a separate parse step. Using assertSqlCanBeParsedAndDeparsed this is not needed anymore.

**Pull request merged**